File: JS Essentials/4.0 For Loops/09. Left and Right Sum.js
function leftAndRightSum(arg) {
    let loop = arg.shift()
    let leftSum = arg.splice(0, loop).reduce((a, b) => +a + +b)
    let rightSum = arg.reduce((a, b) => +a + +b)

    return leftSum == rightSum ? `Yes, sum = ${leftSum}` : `No, diff = ${Math.abs(leftSum - rightSum)}`
}

// console.log(leftAndRightSum([2, 10, 90, 60, 40]))

File: Python Essentials/4.0 For Loops/09. Left and Right Sum.py
loop = int(input())
num_list = [int(input()) for _ in range(loop * 2)]
left_sum = sum(num_list[0:loop])
right_sum = sum(num_list[loop:])

print(f'Yes, sum = {left_sum}' if left_sum == right_sum else f'No, diff = {abs(left_sum - right_sum)}')


'''
2
10
90
60
40
#Yes, sum = 100
'''

